diff -rbU3 fortran/asc2eph.f fortran.new/asc2eph.f
--- fortran/asc2eph.f	2015-03-28 15:06:27.779367306 -0700
+++ fortran.new/asc2eph.f	2015-03-28 15:06:18.167812647 -0700
@@ -71,7 +71,7 @@
 C *****  The user must choose one of the following statements  *****
 C            ( Usually NRECL = 4 is used on Unix platforms)
 
-C      PARAMETER ( NRECL = 4 )
+      PARAMETER ( NRECL = 4 )
 C      PARAMETER ( NRECL = 1 )
 
       INTEGER OLDMAX
diff -rbU3 fortran/jplsubs.f fortran.new/jplsubs.f
--- fortran/jplsubs.f	2015-03-28 14:29:50.512729528 -0700
+++ fortran.new/jplsubs.f	2015-03-28 14:36:17.761503557 -0700
@@ -514,6 +514,7 @@
 C        CALL FSIZER1(NRECL,KSIZE,NRFILE,NAMFIL)
 C        CALL FSIZER2(NRECL,KSIZE,NRFILE,NAMFIL)
 C        CALL FSIZER3(NRECL,KSIZE,NRFILE,NAMFIL)
+      CALL FSIZER4(NRECL,KSIZE,NRFILE,NAMFIL)

       IF(NRECL .EQ. 0) WRITE(*,*)'  ***** FSIZER IS NOT WORKING *****'

