Only in Cdist.new: .Cdist.is_patched
diff -U3 -r Cdist/eph_manager.c Cdist.new/eph_manager.c
--- Cdist/eph_manager.c	2011-02-03 11:37:12.000000000 -0800
+++ Cdist.new/eph_manager.c	2021-10-12 20:32:56.232579302 -0700
@@ -234,6 +234,10 @@
             break;
          case 403: case 405:
          case 421:
+         case 430:
+         case 431:
+         case 440:
+         case 441:
             RECORD_LENGTH = 8144;
             break;
          case 404: case 406:
@@ -321,6 +325,7 @@
    if (EPHFILE)
    {
       error =  (short int) fclose (EPHFILE);
+      EPHFILE = NULL; // new line, reset pointer
       free (BUFFER);
    }
    return error;
Only in Cdist.new: header.440
Only in Cdist.new: linux_m13000p17000.441
Only in Cdist.new: linux_p1550p2650.440
Only in Cdist.new: Makefile
diff -U3 -r Cdist/novas.c Cdist.new/novas.c
--- Cdist/novas.c	2011-03-22 08:43:29.000000000 -0700
+++ Cdist.new/novas.c	2021-10-12 20:43:30.634010238 -0700
@@ -3128,7 +3128,7 @@
    For mean sidereal time, subtract the equation of the equinoxes.
 */
 
-         ha_eq -= (eqeq / 240.0);
+         ha_eq -= (eqeq / 3600.0);
 
          ha_eq = fmod (ha_eq, 360.0) / 15.0;
          if (ha_eq < 0.0)
@@ -7969,7 +7969,7 @@
    the last value of 'n_pts'.
 */
 
-   del_n_pts = abs (n_pts - last_n_pts);
+   del_n_pts = labs (n_pts - last_n_pts);
 
 /*
    Allocate memory for the 't' and 'ra' arrays.
@@ -8032,7 +8032,7 @@
 */
 
    del_index = index_rec - last_index_rec;
-   abs_del_index = abs (del_index);
+   abs_del_index = labs (del_index);
 
 /*
    Determine the file read strategy.
@@ -8069,7 +8069,7 @@
 
     else if ((abs_del_index <= n_pts) && (del_n_pts == 0))
    {
-      n_swap = abs (n_pts - abs_del_index);
+      n_swap = labs (n_pts - abs_del_index);
       n_read = abs_del_index;
 
 /*
diff -U3 -r Cdist/readeph0.c Cdist.new/readeph0.c
--- Cdist/readeph0.c	2011-01-21 06:48:11.000000000 -0800
+++ Cdist.new/readeph0.c	2019-08-12 23:38:31.297605651 -0700
@@ -94,9 +94,9 @@
    formal parameter" compiler warnings.
 */
 
-   mp;
-   name;
-   jd;
+   (void)mp;
+   (void)name;
+   (void)jd;
 
    pv = (double *) malloc (6L * sizeof (double));
 
Only in Cdist.new: testpo.440
